package/MDAnalysis/analysis/align.py

```diff
@@ -1028,34 +1028,29 @@ def get_matching_atoms(ag1, ag2, tol_mass=0.1, strict=False):
 
     if ag1.n_atoms != ag2.n_atoms:
         if ag1.n_residues != ag2.n_residues:
-            errmsg = "Reference and trajectory atom selections do not contain "
-            "the same number of atoms: \n"
-            "atoms:    N_ref={0}, N_traj={1}\n"
-            "and also not the same number of residues:\n"
-            "residues: N_ref={2}, N_traj={3}\n"
-            "\n"
-            "(More details can be found in the log file "
-            "which can be enabled with 'MDAnalysis.start_logging()')".format(
-                ag1.n_atoms, ag2.n_atoms,
-                ag1.n_residues, ag2.n_residues)
-            dbgmsg = "mismatched residue numbers\n" + \
-                "\n".join(["{0} | {1}" for r1, r2 in
-                           zip_longest(ag1.resids, ag2.resids)])
+            errmsg = ("Reference and trajectory atom selections do not contain "
+                      "the same number of atoms: \n"
+                      "atoms:    N_ref={0}, N_traj={1}\n"
+                      "and also not the same number of residues:\n"
+                      "residues: N_ref={2}, N_traj={3}").format(
+                          ag1.n_atoms, ag2.n_atoms,
+                          ag1.n_residues, ag2.n_residues)
             logger.error(errmsg)
-            logger.debug(dbgmsg)
             raise SelectionError(errmsg)
         else:
             msg = ("Reference and trajectory atom selections do not contain "
                    "the same number of atoms: \n"
                    "atoms:    N_ref={0}, N_traj={1}").format(
-                ag1.n_atoms, ag2.n_atoms)
+                       ag1.n_atoms, ag2.n_atoms)
             if strict:
+                logger.error(msg)
                 raise SelectionError(msg)
 
             # continue with trying to creating a valid selection
```

Comment:
`creating` -> `create` typo, but shouldn't kill the messenger for cleaning up other parts of the module

Reply:
fixed